admin/tests/acceptance/authenticated/users/get_test.js

```diff
@@ -56,10 +68,13 @@ module('Acceptance | authenticated/users/get', function (hooks) {
     const userNavigation = within(screen.getByLabelText("Navigation de la section détails d'un utilisateur"));
     assert.dom(userNavigation.getByRole('link', { name: 'Détails' })).exists();
     assert.dom(userNavigation.getByRole('link', { name: 'Profil' })).exists();
-    assert.dom(userNavigation.getByRole('link', { name: 'Participations' })).exists();
+    assert.dom(userNavigation.getByRole('link', { name: `Participations (${user.participations.length})` })).exists();
```

Comment:
```suggestion
    const expectedParticipationLength = 1;
    assert.dom(userNavigation.getByRole('link', { name: `Participations (expectedParticipationsLength)` })).exists();
```

**Question**: Est-ce que dans les tests quand on veut tester une length on la met pas en dur plutôt que de faire un `.length` qui relève de l'implémentation ?

Reply:
Les nombres ont été affichés en dur.